dot/rpc/subscription/websocket_test.go

```diff
@@ -21,12 +21,59 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestWSConn_HandleComm(t *testing.T) {
+func TestWSConn_CheckWebsocketInvalidData(t *testing.T) {
```

Comment:
I am not super sure, but I think if you move this test after `TestWSConn_HandleConn`, git would understand the differ better.